feed.xml

```diff
@@ -0,0 +1,28 @@
+---
+---
+<?xml version="1.0" encoding="utf-8"?>
+<feed xmlns="http://www.w3.org/2005/Atom">
+  <title type="text" xml:lang="en">{{ site.name }}</title>
+  <link type="application/atom+xml" href="atom_feed_url" rel="self"/>
+  <link type="text/html" href="home_url_canonical" rel="alternate"/>
+  <updated>{{ site.time | date_to_xmlschema }}</updated>
+  <id>{{ site.url }}/feed.xml</id>
+
+  {% assign latest_events = site.data.conferences | reverse | slice: 0, 30 %}
```

Comment:
Why wouldn't we want to have all events here? Just for performance reasons?

Reply:
There are currently 363 events. Not great for performance but also unnecessary IMO given how RSS readers work i.e. polling every day or so, if not more frequently, and caching the result

Reply:
But you would anyway only show the ones which actually have an announcement date, no?

Reply:
Yes but this will grow over time. Or someone might decide to backfill the dates in the YAML file which will unleash them.

I think it's good practice to limit the number of items for performance but also for being a good internet citizen. It's is the convention across the internet with RSS. By way of a comparison, the Jekyll feed plugin by default limits the number of blog posts to 10. 

Another way of looking at this is what behaviour you'd like to see when first subscribing to the RSS feed. Would you like to see hundreds of items? Or just the latest 10, 20 or 30? It's this user experience that I'm coming from (as well as performance).

Some readers will only show the latest articles (i.e. newer than 60 days), but asking readers to parse hundreds of heavily outdated content seems like poor form to me.

Reply:
ps. changed to a more generous 50 in https://github.com/ruby-conferences/ruby-conferences.github.io/pull/704/commits/fd8fe6d98415113250cf00d179937c98cb593fe8

Reply:
Fair enough. I guess I was looking at it from the "article/blog post" perspective, where you'd want to see all articles when you subscribe to a feed.

Reply:
This is my point – that's not how RSS for blogs works. 

Most blog publishing systems I know of limit the articles in an RSS feed. Wordpress defaults to 10, Ghost defaults to 15, Jekyll to 10 etc.